Library/Formula/wordgrinder.rb

```diff
@@ -0,0 +1,15 @@
+class Wordgrinder < Formula
+  homepage "http://wordgrinder.sourceforge.net"
+  url "https://downloads.sourceforge.net/project/wordgrinder/wordgrinder/wordgrinder-0.5.2.tar.bz2"
+  sha1 "4bcc600fbd110d9d87961feab15045b9b518b2db"
+
+  depends_on "lua"
+  depends_on "ncurses" => :build
+
+  def install
+    system "make"
```

Comment:
Let's we use `make install` and submit a patch to fix the GNU assumptions? Thanks.